Core.DomainServices/SSO/StsBrugerInfo.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace Core.DomainServices.SSO
+{
+    public class StsBrugerInfo
```

Comment:
StsBrugerInfo? Skulle det ikke være StsUserInfo?

Reply:
Denne er også gældende andre steder med dansk

Reply:
Ja, danglish delen skyldes at typen i Sts hedder "Bruger" og for at vise dette tilhørsforhold i Kitos vælges navnene som sådan